test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -687,8 +687,8 @@ TEST_F(AtenXlaTensorTest, TestIntegerAdd) {
           torch::randint(0, 63, {2, 2}, torch::TensorOptions(type));
       torch::Tensor b =
           torch::randint(0, 63, {2, 2}, torch::TensorOptions(type));
-      at::Scalar one =
-          at::isIntegralType(type) ? at::Scalar(int64_t(1)) : at::Scalar(1.0);
+      torch::Scalar one =
+          isIntegralType(type) ? torch::Scalar(int64_t(1)) : torch::Scalar(1.0);
```

Comment:
Btw I think ? 1 : 1.0` might work here?

Reply:
Both sides of a ternary needs to have same type 😉 
You really need to force int64_t otherwise you will likely get an ambiguous constructor error.

Reply:
Aha right!

Reply:
Surprisingly when I do `torch::Scalar(1)` and `torch::Scalar(1.0)` and print their types I get
`Long` and `Double`. It seems like torch is able to assign the right type to scalar by passing different type of constant. I am fine with making this change